mepo.d/cmdline/parser.py

```diff
@@ -80,6 +87,13 @@ def __clone(self):
             nargs = '?',
             default = None,
             help = 'Configuration file (ignored if init already called)')
+        clone.add_argument(
+            '--style',
+            metavar = 'style-type',
+            nargs = '?',
+            default = 'prefix',
+            choices = ['naked', 'prefix','postfix'],
```

Comment:
I amusingly suggested to the requestor that we allow alternate spellings for these:  Tom|Matt|Arlindo.